src/Redistribution.sol

```diff
@@ -141,41 +141,41 @@ contract Redistribution is AccessControl, Pausable {
     // Reveals for the current round.
     Reveal[] public currentReveals;
 
-    // Role allowed to pause.
-    bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
-
     // Maximum value of the keccack256 hash.
-    bytes32 MaxH = bytes32(0x00000000000000000000000000000000ffffffffffffffffffffffffffffffff);
+    bytes32 private MaxH = bytes32(0x00000000000000000000000000000000ffffffffffffffffffffffffffffffff);
```

Comment:
Shouldn't this also be `constant` ?

Reply:
yes absolutely, cleaned and added in latest commit.